xdsl/dialects/test.py

```diff
@@ -34,7 +36,7 @@ class TestOp(IRDLOperation):
 
 
 @irdl_op_definition
-class TestTermOp(TestOp):
+class TestTermOp(IRDLOperation):
```

Comment:
@math-fehr it seems that this was broken before. This would only collect the constraints of the parent op for verification and not the ones specified within this class. Do we want it to be that way? Or did I just bandage over an IRDL bug?

Reply:
Seems like a :bug: to me yes!

Reply:
opened #1312